dep_updates.md

```diff
@@ -20,7 +20,9 @@ After the automated dependency update PRs are merged (via script, usually - see
 
 ## For Stage and Production Environments
 
-You do NOT need to create a github tagged release *issue* such as https://github.com/LD4P/sinopia_editor/issues/2755 for dependency updates.
+Create a github tagged release *issue* such as https://github.com/LD4P/sinopia_editor/issues/2755 for dependency updates. The rest of this document supplements the instructions in the ticket.
```

Comment:
personally, i probably will keep using the checklist ticket for sinopia dep updates, since i like having the checkboxes to keep track of what's been done.  but i thought the point of this new dep updates doc was to keep people from having to use the checklist?  i might be misremembering/misunderstanding.  @ndushay?

Reply:
I will NOT use the tagged release issue.   It has a bunch of steps re: the product owner and other stuff that is irrelevant.   

Maybe these instructions need to be edited to suggest you can do a tagged release issue OR just follow the instructions here.

Reply:
Notice that this PR changes ticket template to indicate that the Release notes step can be skipped for dependency updates.